cmd/api/main.go

```diff
@@ -136,7 +140,29 @@ func responseJSON(w http.ResponseWriter, status int, body any) {
 	}
 }
 
-/* Verifies with all fields are correctly filled and update the book in the db. */
+/* Set 'true' to column 'archived' in the database */
+func archiveStatusBook(w http.ResponseWriter, r *http.Request) {
+	id, err := isolateId(w, r)
+	if err != nil {
+		return
+	}
+	archived := true //THIS FUNCTION IS READY TO BE IMPROVED TO ALLOW RESTORING BOOKS FROM ARCHIVE. JUST FIX THE ROUTING!
```

Comment:
Can you explain this comment?

Reply:
Sure! On trello:
"Should we ofer a way of re-activate an archived book?
Lets not worry about it now, create a ticket here in Trello so we can add this functionality later."

I was not thinking on the "re-activate" tool, but, looking at the code I've made for "archiving" tool, I've noticed that it will be pretty easy to give a few more steps and let the whole thing ready. So I commented to don't forget it.

Reply:
Buuut, to allow me to continue, I would need your definition of the proper http method responsible for DEarchivate books.

Reply:
We would probably want to create a route for that. It's not a functionality that should be used often and it's probably an "admin" use case.